Shared/IISFunctions/Get-IISModules.ps1

```diff
@@ -48,7 +84,9 @@ function Get-IISModules {
             [CmdletBinding()]
             param(
                 [Parameter(Mandatory = $true)]
-                [object[]]$Modules
+                [object[]]$Modules,
+                [Parameter(Mandatory = $false)]
+                [string[]]$ModulesToSkip
```

Comment:
Let's not do it like this. Keep `GetModulePath` the same, just filter out the modules that we don't want prior to passing to `GetModulePath`, since we aren't returning them at all anyways. When we do the filter, just also add a debug note about it.